src/python_routing_v01/compute_nhd_routing_SingleSeg.py

```diff
@@ -453,22 +615,39 @@ def main():
         print("reach organization complete")
     if showtiming:
         print("... in %s seconds." % (time.time() - start_time))
-
-    if showtiming:
         start_time = time.time()
+
     connections = supernetwork_values[0]
 
-    flowdepthvel = {
-        connection: {
-            "flow": {"prev": 0, "curr": 0},
-            "depth": {"prev": 0, "curr": 0},
-            "vel": {"prev": 0, "curr": 0},
-            "qlat": {"prev": 0, "curr": 0},
-        }
-        for connection in connections
-    }
-
-    parallelcompute = False
+    flowdepthvel = {connection: {'qlatval': []
+        , 'time': []
+        , 'flowval': []
+        , 'depthval': []
+        , 'velval': []} for connection in connections}
+
+
+    # Lateral flow
+    ## test 1. Take lateral flow from wrf-hydro output from Pocono Basin
+    ql_input_folder = os.path.join(
+        root, r"test/input/geo/PoconoSampleData2/Pocono_ql_testsamp1_nwm_mc.csv"
+    )
+    ql = pd.read_csv(ql_input_folder, index_col=0)
+
+    for index, row in ql.iterrows():
+        # print(index, row.to_numpy())
+        flowdepthvel[index]["qlatval"] = row.to_numpy().tolist()
```

Comment:
Replace `row.to_numpy().tolist()` with `row.to_list()`

Reply:
We saw this and we'll address it in the final PR. 
@jdhondia-noaa

Reply:
Updated in [80765c6](https://github.com/NOAA-OWP/t-route/pull/77/commits/80765c6c259688f461298076fa9eff4753a9307f)